ds-prep-python-sets/main.py

```diff
@@ -0,0 +1,48 @@
+# 3
+trail_mix = set({'m&ms', 'peanuts', 'raisins'})
+
+# 4
+print(trail_mix)
+
+# 6
+print('cashews is in trail_mix: {}'.format('cashew' in trail_mix))
+
+# 7
+print('peanuts is in trail_mix: {}'.format('peanuts' in trail_mix))
+
+# 9
+trail_mix.add('pretzels')
+
+# 10
+trail_mix.update(['peanuts', 'banana chips', 'bits of jerky'])
```

Comment:
While this does technically follow directions, the whole point of this exercise is to practice with `set`s, not `list`s, so I'd suggest only using `set`s.